src/templates/best-cards/generic/animalism.html

```diff
@@ -25,8 +25,8 @@ <h1><i>i</i> Animalism</h1>
 
 {% call layout.card_column("Cats' Guidance", decks=222, copies="2-5") %}
 {% trans trimmed %}
-A versatile defence tool providing either intercept or an unlock effect.
-Not as suprising as a wake, but being unlocked can be better.
+A versatile defense tool providing either intercept or an unlock effect.
```

Comment:
Eheh, you're not the first to point that out. `Defence` is en-GB and `Defense` typically en-US.
I'm all for switching to US as it seems to bother many non-native speakers, but in this case we'll need to fix it in all files 😋

Reply:
We can also leave it that way. It's understandable and not disrupting the read. If the non-native speakers are all french, solution will be provided by the translation of the page :)

Reply:
As you wish, I don't mind one way or the other - just try to keep it consistent.